### PR TITLE
Added PLM and Material Management ontology modules.

### PR DESCRIPTION
**Pull request by marekmeyer**

### CODE REVIEW

**Comment by jane-doe** [comment]
Can one of the admins verify this patch?

File: NTO/Documents/attributes/documentPart.ttl [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit.Documents:documentPart
+	a owl:DatatypeProperty;
+	rdfs:subPropertyOf ogit:Attribute;
+	rdfs:label "documentPart";
+	dcterms:description "Contains the part of a document.";
```
Comment (KaushikGondaliya) [inline NTO/Documents/attributes/documentPart.ttl:11]: "Contains the part of a document."

What does this mean and what information stored here? If it is part of documents and then its already document (and probably entity instead attribute?)

File: NTO/Documents/attributes/documentNumber.ttl [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit.Documents:documentNumber
+	a owl:DatatypeProperty;
+	rdfs:subPropertyOf ogit:Attribute;
+	rdfs:label "documentNumber";
+	dcterms:description "Contains the number of a document.";
+	dcterms:valid "start=2020-04-16;";
```
Comment (KaushikGondaliya) [inline NTO/Documents/attributes/documentNumber.ttl:12]: its 2021 now :P

It's the same in some other files as well.

**Review by KaushikGondaliya — CHANGES_REQUESTED** [review]

File: NTO/Documents/attributes/documentPart.ttl (hunk 1 above)
Comment (marekmeyer) [reply NTO/Documents/attributes/documentPart.ttl:11]: This attribute is required for document info records from SAP. "Part of document" is relevant for larger original documents, which are split into multiple parts. See: https://blog.sap-press.com/how-to-create-a-document-info-record-in-sap-s4hana

File: NTO/Documents/attributes/documentNumber.ttl (hunk 2 above)
Comment (marekmeyer) [reply NTO/Documents/attributes/documentNumber.ttl]: Fixed for all four attributes.

**Comment by marekmeyer** [comment]
@KaushikGondaliya can you please review the changes.

**Review by KaushikGondaliya — APPROVED** [review]

**Pull request merged**